src/containers/BooksForm.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook } from '../actions/index';
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: 'Action',
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange() {
+    this.setState({
+      title: document.querySelector('input').value,
+      category: document.querySelector('select').value,
+    });
+  }
+
+  render() {
+    const { addBook } = this.props;
+
+    const handleSubmit = e => {
+      e.preventDefault();
+      const params = { ...this.state, id: Math.random() };
+      document.querySelector('input').value = '';
```

Comment:
Hi team, while this works fine, it is not the best way to write react for everytime you called document.querySelector you are beating the idea of react. handleChange is called on component when those components are clicked an event is fired. `event.target.value` should do what `document.queryselector` is doing